qt/aqt/forms/preferences.ui

```diff
@@ -103,6 +103,13 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="load_last_loaded_profile">
+         <property name="text">
+          <string notr="true">Load last loaded profile by default</string>
```

Comment:
Because of https://forums.ankiweb.net/t/set-html-editor-as-a-default-editor-instead-of-visual-editor/20988/7, perhaps it would be better to start off by switching from the old functionality to the new functionality, without an option to change back to the old way. Hopefully, people will like the new behaviour, and we won't need another option. If that turns out to be wrong, we can always add an option later.

(if/when we do so, we would need to make that string translatable by putting it in an .ftl file)